tests/testthat/test-undesirable_operator_linter.R

```diff
@@ -29,3 +29,15 @@ test_that("undesirable_operator_linter handles infixes correctly", {
     undesirable_operator_linter(list("%%" = NA))
   )
 })
+
+test_that("undesirable_operator_linter vectorizes messages", {
+  expect_lint(
+    "x <<- c(pkg:::foo, bar %oo% baz)",
+    list(
+      rex::rex("`<<-` is undesirable. It assigns"),
+      rex::rex("`:::` is undesirable. It accesses"),
+      rex::rex("`%oo%` is undesirable.", end)
+    ),
+    undesirable_operator_linter(modify_defaults(default_undesirable_operators, "%oo%" = NA))
```

Comment:
Maybe we should also put the `modify_defaults()` in the docs for `undesirable_{operator,function}_linter()`. Follow-up?

Reply:
#1306